packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.ts

```diff
@@ -56,11 +68,16 @@ export type UserStorageControllerState = {
    * Loading state for the profile syncing update
    */
   isProfileSyncingUpdateLoading: boolean;
+  /**
+   * Loading state for the account syncing update
+   */
+  isUserStorageAccountSyncingInProgress: boolean;
```

Comment:
What do we need this loading state for in the UI.
(not against if need to attach this to the UI in future)

Reply:
@Prithpal-Sooriya I'm not even sure, I added it for the sake of maybe product asking for a loading indicator, but I didn't hear back on this, so maybe best to delete this (and maybe re-add later)? WDYT?